include/config.h

```diff
@@ -12,13 +12,17 @@ enum mako_binding_action {
 	MAKO_BINDING_DISMISS,
 	MAKO_BINDING_DISMISS_GROUP,
 	MAKO_BINDING_DISMISS_ALL,
+	MAKO_BINDING_INVOKE_ACTION,
 	MAKO_BINDING_INVOKE_DEFAULT_ACTION,
 	MAKO_BINDING_EXEC,
 };
 
 struct mako_binding {
 	enum mako_binding_action action;
-	char *command; // for MAKO_BINDING_EXEC
+	union {
+		char *command;     // for MAKO_BINDING_EXEC
+		char *action_name; // for MAKO_BINDING_ACTION
+	};
```

Comment:
This is up for discussion, its saves a pointer but makes `free(binding->command)` a bit weird when used for `MAKO_BINDING_INVOKE_ACTION`.

Alternatives:
- rename to something generic like `data`
- just add a 2nd pointer
- reuse the `command` pointer

Reply:
I'd prefer to just use two separate fields without `union`.

Reply:
changed